sections/main-product.liquid

```diff
@@ -718,7 +718,7 @@
       "settings": [
         {
           "type": "text",
-          "id": "link_label",
+          "id": "text",
```

Comment:
I think we can leave this unchanged as we show the name from the resource attached which should be unique across different blocks as shown in the screenshot below. I think it is less likely there will be different popups linking to the same page resource.

Name comes from the `About us` resource attached.
<img width="1717" alt="Screen Shot 2021-08-11 at 8 52 05 AM" src="https://user-images.githubusercontent.com/22102095/129032001-8ba74e0c-1c8f-4a76-85e5-9488cc4806dc.png">

Reply:
Since you can have more than 1 I think it might be beneficial here to have it. So you can see which link/popup is where: 
![](https://screenshot.click/21-08-kl5l7-rhwt1.png)

Reply:
Good point. So it relates more to what you're actually seeing in preview 👌